index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="stylesheet" href="main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#"><h2>Welcome</h2></a>
+        <img src="images/more.png" alt="more" />
+      </nav>
+    </header>
+    <main>
+      <section class="main-title">
+        <div>
+          <h1 class="main-header">Hey there. I'm Nati</h1>
+          <h1 class="highligh-title">I'm a software developer</h1>
```

Comment:
- [ ] Please replace the `<h1>` tag in line 22 with a `<h2>` tag because semantically you are only allowed to use one `<h1>` tag per page or you can do it that way:

```
<h1 class="main-header">
     Hey there. I'm Nati <br>
     I'm a software developer
</h1>
```

Reply:
@Omar-Muhamad Thank you for your feedback. My question would be, if I use the code example you sent me here, how would I add different CSS for the second line after the first line or after the <be>?

Reply:
Hello @NatiDeme, like the first suggestion use `<h2>` instead.

Have a great day.